src/main/java/sirius/web/controller/BasicController.java

```diff
@@ -183,7 +183,7 @@ public void onError(WebContext webContext, @Nonnull HandledException error) {
     }
 
     @Override
-    public void onJsonError(WebContext webContext, HandledException error) {
+    public void onApiError(WebContext webContext, HandledException error, Format format) {
```

Comment:
Imho this is / could be a breaking change and the PR should be marked as such

Reply:
done